api/data/pipeline.py

```diff
@@ -79,15 +80,32 @@ def main():
     # Merging Clinvar
     clinvar = clinvar_data.copy()[["Name(clinvar)",
                                    "Germline classification(clinvar)",
-                                   "Accession(clinvar)"]]
+                                   "Accession(clinvar)",
+                                   "GRCh38Location(clinvar)"]]
+
     clinvar["VariantOnTranscript/DNA"] = (clinvar["Name(clinvar)"].
                                           apply(from_clinvar_name_to_cdna_position))
-
     main_frame = pd.merge(main_frame,
                           clinvar,
                           how="outer",
                           on=["VariantOnTranscript/DNA"]).drop("Name(clinvar)", axis=1)
 
+    start_merge = pd.merge(main_frame,
+                           clinvar,
+                           how="outer",
+                           left_on="position_g_start",
+                           right_on="GRCh38Location(clinvar)").drop(["Name(clinvar)", "GRCh38Location(clinvar)"],
+                                                                    axis=1)
+
+    end_merge = pd.merge(main_frame,
+                         clinvar,
+                         how="outer",
+                         left_on="position_g_end",
+                         right_on="GRCh38Location(clinvar)").drop(["Name(clinvar)", "GRCh38Location(clinvar)"],
+                                                                  axis=1)
+
+    main_frame = start_merge.combine_first(end_merge)
+
```

Comment:
Make it as single function that gets lovd tables and clinvar table and returns merged data. If you see more parameters are required, then add them.

Reply:
Assume that lovd and clinvar wasn't changed.

Reply:
**Assume that lovd and clinvar wasn't changed.**

Reply:
Change columns name after/during merge